dropwizard-health/src/test/java/io/dropwizard/health/response/JsonHealthResponseProviderTest.java

```diff
@@ -127,6 +129,8 @@ void shouldThrowExceptionWhenJsonProcessorExceptionOccurs() throws IOException {
     }
 
     private String fixture(final String filename) throws IOException {
-        return new String(getClass().getResourceAsStream(filename).readAllBytes(), UTF_8);
+        try (InputStream is = getClass().getResourceAsStream(filename)) {
+            return new String(requireNonNull(is).readAllBytes(), UTF_8);
```

Comment:
@rhowe `is.readAllBytes()` will already fail with a `NullPointerException` which is exactly what happens when wrapping it in `Objects.requireNonNull(...)`.

Reply:
@joschi I already saw this in IntelliJ as a suggested fix. When not doing this, the IntelliJ linter will complain that the variable could be `null`. However, if wrapped in `requireNonNull`, the returned object can't be `null` because of the NPE thrown in the method.

Reply:
Yeah, I threw it in for exactly that reason - the IntelliJ linter being dumb.

It isn't a great reason, to be fair. Can take it out?

Reply:
I personally don't have a strong opinion on either solution. As I think most developers use IntelliJ, we could leave it there to make the linter happy.